onnx/test/test_external_data.py

```diff
@@ -238,7 +238,7 @@ def test_save_external_invalid_single_file_data_and_check(
                 traversal_external_data_dir, "tensors.bin"
             )
         else:
-            traversal_external_data_location = "../invlid_external_data/tensors.bin"
+            traversal_external_data_location = "invlid_external_data/tensors.bin"
```

Comment:
We should keep this path because it is intentional to be this. Instead, you may find a way to create a model that contains this path, by perhaps having a test version of the save external data function that does not enforce the traversal check.